turbo.json

```diff
@@ -3,38 +3,27 @@
     "pipeline": {
         "build": {
             "dependsOn": [
-                "compile:docs",
                 "compile:js",
                 "compile:typedefs",
                 "test:lint",
                 "test:prettier",
-                "test:typecheck",
-                "test:unit:browser",
-                "test:unit:node",
-                "test:live-with-test-validator",
                 "test:treeshakability:browser",
                 "test:treeshakability:native",
-                "test:treeshakability:node"
+                "test:treeshakability:node",
+                "test:typecheck",
+                "test:unit:browser",
+                "test:unit:node"
             ],
-            "outputs": ["dist/**", "lib/**"]
-        },
-        "clean": {
-            "outputs": ["dist/**", "lib/**"]
-        },
-        "compile:docs": {
-            "dependsOn": ["clean"],
-            "inputs": ["src/**"],
-            "outputs": ["doc/**"]
+            "outputs": ["dist/**"]
         },
         "compile:js": {
-            "dependsOn": ["clean", "^compile:js"],
```

Comment:
Removing `"dependsOn": ["^compile:js"]` is OK because of the change we made in #2374.